sale_order_margin_percent/models/sale_order_margin.py

```diff
@@ -0,0 +1,26 @@
+##############################################################################
+#
+#    OmniaSolutions, Open Source Management Solution
+#    2010-2018 OmniaSolutions (<http://www.omniasolutions.eu>).
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+##############################################################################
+
+from odoo import models, api, fields
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    percent = fields.Float(
+        string='Percent',
+        compute='_compute_percent')
```

Comment:
@OmniaGit IMO add `default=0.0` or `order.percent= 0.0` [here](https://github.com/OCA/sale-workflow/pull/692/files#diff-1f798e0f0bf1d10e29f35ab319481572R24 )

Reply:
Being a compute, it doesn't need a default. It's automatically reset to that default at the beginning of the compute method.